src/utils/routes.js

```diff
@@ -9,30 +9,30 @@ export const UPDATE_PROFILE = "/users/profile/?id=5deab2193026f30ec826a0cc"; //
 export const VIEW_USERS = "/users?sortBy=createdAt&sortType=asc"; // GET
 
 /** EVENTS MODULE : PARTICIPANTS **/
-export const REGISTER_PARTICIPANT = "/events/register_part"; // POST
-export const LOGIN_PARTICIPANT = "/events/part_login"; // POST
-export const REGISTER_FOR_EVENT = "/events/register_in_event"; // POST
+export const REGISTER_PARTICIPANT = "/events/participants/"; // POST
+export const LOGIN_PARTICIPANT = "/events/participants/login"; // POST
+export const REGISTER_FOR_EVENT = "/events/register"; // POST
 export const UPDATE_PARTICIPANT =
-	"/events/update_part/5e6a67b4890fda16986ae843"; // PUT
-export const GET_PARTICIPANT_DETAILS = "/events/part_data"; // GET
+  "/events/participants/5e6a67b4890fda16986ae843"; // PUT
+export const GET_PARTICIPANT_DETAILS = "/events/participants/profile"; // GET
 // ?participantId=5e6bc1dec3988900640bd922
-export const GET_PARTICIPANTS = "/events/get_part"; // GET
+export const GET_PARTICIPANTS = "/events/participants"; // GET
 // ?query=ritik&eventId=5e6dfb7caad4441a9ceb5b2e
 
 /** EVENTS MODULE : EVENTS **/
-export const GET_EVENTS = "/events/get_events"; // GET
-export const GET_EVENT = "/events/get_events/"; // GET
-export const ADD_EVENT = "/events/add_event"; // POST
-export const CHANGE_EVENT_CODE = "/events/change_event_code"; // POST
-export const TOGGLE_REGISTRATION = "/events/event_regist_open"; // POST
-export const UPDATE_EVENT = "/events/update_event"; // PUT
-export const DELETE_EVENT = "/events/delete_event"; // DELETE
+export const GET_EVENTS = "/events/"; // GET
+export const GET_EVENT = "/events/"; // GET
```

Comment:
This is a wrong route
```suggestion
export const GET_EVENT = "/events"; // GET
```
Due to this, the event details were not being fetched correctly

Reply:
Okay